maven-surefire-plugin/src/site/resources/xsd/failsafe-summary.xsd

```diff
@@ -1 +1 @@
~ under the License.
-->
<xsd:schema version="1.0" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
    <xsd:element name="failsafe-summary">
        <xsd:complexType>
            <xsd:sequence>
                <xsd:element name="completed" type="xsd:int"/>
                <xsd:element name="errors" type="xsd:int"/>
                <xsd:element name="failures" type="xsd:int"/>
                <xsd:element name="skipped" type="xsd:int"/>
                <xsd:element name="failureMessage" type="xsd:string" nillable="true"/>
            </xsd:sequence>
            <xsd:attribute name="result" type="errorType" use="optional"/>
            <xsd:attribute name="timeout" type="xsd:boolean" use="required"/>
        </xsd:complexType>
    </xsd:element>
    <xsd:simpleType name="errorType">
        <xsd:restriction base="xsd:string">
            <xsd:enumeration id="FAILURE" value="255"/>
            <xsd:enumeration id="NO_TESTS" value="254"/>
        </xsd:restriction>
    </xsd:simpleType>
</xsd:schema>
\ No newline at end of file
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<!--
~ Licensed to the Apache Software Foundation (ASF) under one
~ or more contributor license agreements.  See the NOTICE file
~ distributed with this work for additional information
~ regarding copyright ownership.  The ASF licenses this file
~ to you under the Apache License, Version 2.0 (the
~ "License"); you may not use this file except in compliance
~ with the License.  You may obtain a copy of the License at
~
~     http://www.apache.org/licenses/LICENSE-2.0
~
~ Unless required by applicable law or agreed to in writing,
~ software distributed under the License is distributed on an
~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
~ KIND, either express or implied.  See the License for the
~ specific language governing permissions and limitations
~ under the License.
-->
<xsd:schema version="1.0" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
    <xsd:element name="failsafe-summary">
        <xsd:complexType>
            <xsd:sequence>
                <xsd:element name="completed" type="xsd:int"/>
                <xsd:element name="errors" type="xsd:int"/>
                <xsd:element name="failures" type="xsd:int"/>
                <xsd:element name="skipped" type="xsd:int"/>
                <xsd:element name="flakes" type="xsd:int" nillable="true"/>
                <xsd:element name="failureMessage" type="xsd:string" nillable="true"/>
            </xsd:sequence>
            <xsd:attribute name="result" type="errorType" use="optional"/>
            <xsd:attribute name="timeout" type="xsd:boolean" use="required"/>
        </xsd:complexType>
    </xsd:element>
    <xsd:simpleType name="errorType">
        <xsd:restriction base="xsd:string">
            <xsd:enumeration id="FAILURE" value="255"/>
            <xsd:enumeration id="NO_TESTS" value="254"/>
        </xsd:restriction>
    </xsd:simpleType>
</xsd:schema>

```

Comment:
What happended to line ending here?

Reply:
This diff is really hard to understand (apologies). Which one are you referring to?

Reply:
Can you undo a change to this file, something is off here or are you expected to change the schema?!

Reply:
`<xsd:element name="flakes" type="xsd:int" nillable="true"/>`

Given I was changing the shape of the `failsafe-summary.xml`; I thought I would need to modify the .xsd - is this not necessary? Happy to undo this change

Reply:
...right, but nillable and minOccurs are two different things. See https://github.com/apache/maven-surefire/pull/790/files#r1820448168. I consider the usage of nillable non-reasonable. Make it `minOccurs="0"` with a comment.

Reply:
TIL; thanks.